### PR TITLE
Expose around worker in Runtime builder

### PR DESCRIPTION
**Pull request by LucioFranco**

### CODE REVIEW

File: tokio/src/runtime/threadpool/builder.rs [hunk 1]
```diff
@@ -347,18 +381,18 @@ impl Builder {
...
             })
             .custom_park(move |worker_id| {
                 let index = worker_id.to_usize();
 
-                timers[index]
-                    .lock()
-                    .unwrap()
-                    .take()
-                    .unwrap()
+                timers[index].lock().unwrap().take().unwrap()
```
Comment (hawkw) [inline tokio/src/runtime/threadpool/builder.rs]: Was this a rustfmt change?

File: tokio/src/runtime/threadpool/builder.rs [hunk 2]
```diff
@@ -58,6 +58,8 @@ pub struct Builder {
 
     /// The clock to use
     clock: Clock,
+
+    around_worker: Option<Arc<dyn Fn(&threadpool::Worker) + Send + Sync + 'static>>,
```
Comment (hawkw) [inline tokio/src/runtime/threadpool/builder.rs]: Nit: every other field has a doc comment, and this is one of the less self-explanatory fields IMO

File: tokio/src/runtime/threadpool/builder.rs [hunk 3]
```diff
@@ -377,3 +411,13 @@ impl Default for Builder {
         Self::new()
     }
 }
+
+impl fmt::Debug for Builder {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("Builder")
+            .field("threadpool_builder", &self.threadpool_builder)
+            .field("core_threds", &self.core_threads)
```
Comment (hawkw) [inline tokio/src/runtime/threadpool/builder.rs]: typo:
```suggestion
            .field("core_threads", &self.core_threads)
```

File: tokio/src/runtime/threadpool/builder.rs [hunk 4]
```diff
@@ -244,6 +246,35 @@ impl Builder {
...
+    ///     })
+    ///     .build();
+    /// ```
+    ///
+    /// [`Worker::run`]: struct.Worker.html#method.run
+    pub fn around_worker<F>(&mut self, f: F) -> &mut Self
+    where
+        F: Fn(&threadpool::Worker) + Send + Sync + 'static,
+    {
+        self.threadpool_builder.around_worker(f);
```
Comment (carllerche) [inline tokio/src/runtime/threadpool/builder.rs]: This should set `self.around_worker`.

**Comment by carllerche** [comment]
We probably should add a test too.

**Comment by carllerche** [comment]
I'm going to close this PR as I believe #1657 takes over, but thanks for doing the work :+1:

**Pull request closed without merging**